Cargo.toml

```diff
@@ -15,11 +15,11 @@ required-features = ["process"]
 process = ["dprint-core/process"]
 
 [dependencies]
-dprint-core = "0.42.0"
-rustfmt-nightly = { git = "https://github.com/rust-lang/rustfmt" }
-serde = { version = "1.0.88", features = ["derive"] }
+dprint-core = "0.49.1"
+rustfmt-nightly = { git = "https://github.com/rust-lang/rustfmt", rev = "7913f13" }
```

Comment:
Not sure if you know this but you can bump a git dependency that is tracked via the lockfile with:

```shell
cargo update -p rustfmt-nightly
```

Reply:
What benefit does that bring? It seems better to have the revision stored here because it's like a version number. I'm probably missing something.

Reply:
I don't think there is a functional difference.

It gets confusing if you ever look at the lockfile yourself because it will have two revisions in the URL. The first one being the revision that latest HEAD was at the time you last updated.
The second one being the revision you actually want to checkout.

It is like a version number yes but even for crates.io dependencies, the version that is actually used is only in the lockfile. It probably comes down to personal preference really :)